packages/manager/apps/hycu/src/routes/routes.tsx

```diff
@@ -120,6 +120,17 @@ export const Routes: any = [
           },
         },
       },
+      {
+        id: 'edit',
+        path: urls.editPack,
+        ...lazyRouteConfig(() => import('@/pages/edit-pack/EditPack.page')),
+        handle: {
+          tracking: {
+            pageName: 'edit',
```

Comment:
same here, `edit-pack` would be more explicit as you also have `edit-name` route